pkg/transport/transport.go

```diff
@@ -116,7 +117,13 @@ func createRoutingEndpoint(rc *Context, logger Logger) func(w http.ResponseWrite
 		}
 
 		transformer.SetRESTHeaders(w.Header(), header, trailer)
-		fmt.Fprint(w, protojson.Format(rpcResponse))
+		response, err := rc.Encoder.Format(rpcResponse)
+		if err != nil {
+			logger.ErrorContext(r.Context(), jErrors.Details(jErrors.Trace(err)))
+			w.WriteHeader(http.StatusInternalServerError)
+		}
+
+		fmt.Fprint(w, response)
```

Comment:
Avoid using of `fmt` when writing HTTP response body. Use `io.Copy`, `io.WriteString()` or `w.Write()`.